.github/workflows/main.yml

```diff
@@ -1,44 +1,102 @@
-#       sudo wget -O /usr/bin/bin2sgl.Linux https://raw.githubusercontent.com/rogerclarkmelbourne/OpenGD77/master/firmware/tools/bin2sgl.Linux
-#       sudo wget -O /usr/bin/gd-77_firmware_loader.py https://raw.githubusercontent.com/rogerclarkmelbourne/OpenGD77/master/tools/Python/FirmwareLoader/gd-77_firmware_loader.py
-#       sudo chmod +x /usr/bin/bin2sgl /usr/bin/bin2sgl.Linux /usr/bin/gd-77_firmware_loader.py
-    - name: meson-setup
-      run: |
-        cd ${{github.workspace}}
-        meson setup build_linux
-        meson setup --cross-file cross_arm.txt build_arm
-    - name: ninja
-      run: |
-        cd ${{github.workspace}}
-        ninja -C build_linux openrtx_linux -j4
-        ninja -C build_arm openrtx_md3x0_wrap -j4
-        ninja -C build_arm openrtx_mduv3x0_wrap -j4
-        ninja -C build_arm openrtx_gd77_bin -j4
-        ninja -C build_arm openrtx_dm1801_bin -j4
-    - uses: actions/upload-artifact@v2
-      with:
-        name: release-bins
-        path: |
-          ${{github.workspace}}/build_arm/openrtx_*_wrap*
-          ${{github.workspace}}/build_linux/openrtx_linux
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: install-deps
+        run: |
+          sudo apt update
+          sudo apt install git pkg-config build-essential meson libsdl2-dev libreadline-dev dfu-util cmake libusb-1.0-0 libusb-1.0-0-dev
+      - name: build install codec2
+        run: |
+          cd ${{github.workspace}}
+          meson subprojects download
+          cd subprojects/codec2
+          mkdir build_linux
+          cd build_linux
+          cmake ..
+          make
+          sudo make install
+      - name: install miosix
+        run: |
+          wget https://miosix.org/toolchain/MiosixToolchainInstaller.run
+          chmod +x MiosixToolchainInstaller.run
+          sudo sh MiosixToolchainInstaller.run
+          wget https://github.com/v0l/radio_tool/releases/download/v$RADIO_TOOL_VERSION/radio_tool-$RADIO_TOOL_VERSION-Linux.deb
+          sudo dpkg -i radio_tool-$RADIO_TOOL_VERSION-Linux.deb
+      - name: Get opengd77 for wrapping
+        run: |
+          git clone https://github.com/open-ham/OpenGD77.git /tmp/OpenGD77
+      - name: setup meson
+        run: |
+          cd ${{github.workspace}}
+          meson setup build_linux
+          meson setup --cross-file cross_arm.txt build_arm
+      - name: Compile linux
+        run: meson compile -C build_linux openrtx_linux
+      - name: Compile arm targets
+        run: |
+          export PATH=$PATH:/tmp/OpenGD77/tools/Python/FirmwareLoader:/tmp/OpenGD77/firmware/tools
+          echo $PATH
+          meson compile -C build_arm openrtx_md3x0_wrap
+          meson compile -C build_arm openrtx_mduv3x0_wrap
+          meson compile -C build_arm openrtx_md9600_wrap
+          meson compile -C build_arm openrtx_gd77_wrap
```

Comment:
@n1zzo the gd77 wrapping isn't working despite [having these steps satisfied](http://openrtx.org/#/compiling?id=gd-77-and-dm-1801). Can you take a look at the commands that are being run here and the output of the build to shed some light on what may be happening?

Reply:
Not sure what error you are getting but now there isn't the need anymore to have the OpenGD77 binaries in the PATH since they are in the scripts directory but the doc wasn't updated. Soon there should support for SGL files in radio_tool, so we'll move to that for the GD-77 & co. in the future.
You should remove the opengd77 step, the export of that PATH and fix the release-bins wildcard to include sgl files they have a ".sgl" extension, otherwise this PR looks good to me